integration-tests/smoke/automation_test.go

```diff
@@ -143,13 +148,20 @@ func SetupAutomationBasic(t *testing.T, nodeUpgrade bool) {
 				defaultAmountOfUpkeeps,
 				big.NewInt(automationDefaultLinkFunds),
 				automationDefaultUpkeepGasLimit,
+				isLogTrigger,
 			)
 
 			l.Info().Msg("Waiting for all upkeeps to be performed")
 			gom := gomega.NewGomegaWithT(t)
 			gom.Eventually(func(g gomega.Gomega) {
 				// Check if the upkeeps are performing multiple times by analyzing their counters and checking they are greater than 10
 				for i := 0; i < len(upkeepIDs); i++ {
+					// Kick off the upkeep, and count!
+					err := consumers[i].Start()
```

Comment:
This is how we kick off the log trigger and count.

Reply:
Both conditional and log based upkeeps can run e2e pretty stably. I triggered this CI test 5 times, and the `ETH smoke tests automation` passed all of them.